lalrpop/src/lexer/intern_token/mod.rs

```diff
@@ -59,7 +56,10 @@ pub fn compile<W: Write>(
     }
 
     if !contains_skip {
-        rust!(out, r#"(r"^(\s*)", true),"#);
+        #[cfg(feature = "unicode")]
+        rust!(out, r#"(r"\s+", true),"#);
+        #[cfg(not(feature = "unicode"))]
+        rust!(out, r#"(r"[ \t\r\n]+", true),"#);
```

Comment:
For consistency, does it make sense to use `?-u:\s`? https://docs.rs/regex/latest/regex/#unicode

Reply:
Looks better. I'll push a commit later.